src/IRac.cpp

```diff
@@ -2211,19 +2259,28 @@ void IRac::samsung(IRSamsungAc *ac,
 /// @param[in] on The power setting.
 /// @param[in] mode The operation mode setting.
 /// @param[in] degrees The temperature setting in degrees.
+/// @param[in] sensorTemp The room (iFeel) temperature sensor reading in degrees
+///                       Celsius.
 /// @param[in] fan The speed setting for the fan.
 /// @param[in] swingv The vertical swing setting.
+/// @param[in] iFeel Whether to enable iFeel (remote temp) mode on the A/C unit.
 /// @param[in] beep Enable/Disable beeps when receiving IR messages.
 /// @param[in] sleep Nr. of minutes for sleep mode. -1 is Off, >= 0 is on.
 void IRac::sanyo(IRSanyoAc *ac,
                  const bool on, const stdAc::opmode_t mode,
-                 const float degrees, const stdAc::fanspeed_t fan,
-                 const stdAc::swingv_t swingv, const bool beep,
-                 const int16_t sleep) {
+                 const float degrees, const float sensorTemp,
+                 const stdAc::fanspeed_t fan, const stdAc::swingv_t swingv,
+                 const bool iFeel, const bool beep, const int16_t sleep) {
   ac->begin();
   ac->setPower(on);
   ac->setMode(ac->convertMode(mode));
   ac->setTemp(degrees);
+  if (sensorTemp != kNoTempValue) {
+    ac->setSensorTemp(static_cast<uint8_t>(sensorTemp + 0.5));
+  } else {
+    ac->setSensorTemp(degrees);  // Set the sensor temp to the desired temp.
```

Comment:
Change comment to:
`// Set the sensor temp to the desired (normal) temp.` or make it clearer/more obvious.

This confused me for a bit as I read the comment, and not the code. I thought you were doing:
`ac->setSensorTemp(sensorTemp);` here instead of `degrees` based on the comment.

Reply:
Will update as suggested.

*  **Disclamer:** I copied this as-is from what used to be line `2242`. To be honest, I don't know exactly why it was being done _(seems counterintuitive: always sets `desired == actual` so I'd assume the A/C may decide to do nothing in this case)_.
I have left it however, as it was there in the original code. If you know if this can be removed, I'd happily delete this.



On the flip side, I have even debated if not to apply this pattern globally, e.g.:
```c++
stdAc::state_t IRac::cleanState(const stdAc::state_t state) {
    stdAc::state_t result = state;
    // (...)
    if (state.sensorTemp == kNoTempValue) result.sensorTemp = state.degrees;   //< HERE
    return result;
```
But finally decided against it as too risky.